app/scripts/metamask-controller.js

```diff
@@ -4799,17 +4800,17 @@ export default class MetamaskController extends EventEmitter {
    * @param {MessageSender | SnapSender} options.sender - The sender of the messages on this stream.
    * @param {string} [options.subjectType] - The type of the sender, i.e. subject.
    */
-  setupUntrustedCommunication({ connectionStream, sender, subjectType }) {
+  setupUntrustedCommunicationLegacy({ connectionStream, sender, subjectType }) {
```

Comment:
@Gudahtt do you have a suggestion for this function name?

Reply:
renamed to `setupUntrustedCommunicationEip1193` here https://github.com/MetaMask/metamask-extension/pull/25075/commits/ccd10abaa4f0ec1ccc34e552b0f3cc9cddb9f2a8